packages.tsv

```diff
@@ -15,6 +15,13 @@ conda	tmux	2.1--1	tmux -V
 conda	m4	1.4.17--0	test $(echo 'define(a, bbb)a' | m4) = 'bbb'
 conda	samtools	1.3--1	samtools --version
 conda	samtools	1.0--0	samtools --version
+conda	bowtie2	2.2.5--0	(bowtie2 --version 2>&1) > /dev/null
```

Comment:
I don't know if the build fails because of this, but this test seems overly complex to me. A simple `bowtie2 --version` might produce debug output, and we are matching for exit code only anyways.